### DISCUSSION TITLE
Welcome to the exist-webcomponents team!

### DISCUSSION THREAD

**Discussion by joewiz** [eXist-db/exist#4877] [Team Posts]
As discussed on the recent community call and the chat room, the xquery-contributors team has now absorbed members and repos from the exist-webcomponents team. Welcome! We greatly appreciate the efforts of the exist-webcomponents team, in particular @JoernT, in reviving the Dashboard application and dramatically advancing it in preparation for inclusion in a forthcoming release candidate. At this phase, it's more important than ever that we work together as a team to test releases, triage and respond to user-reported issues, and meet the high standards of apps included in the standard eXist distribution. 

In order to ensure the new Dashboard is included with this week's release of eXist 5.0.0-RC, I believe the following steps are needed:

1. Would a team member please merge https://github.com/eXist-db/existdb-dashboard/pull/14? @line-o has already reviewed it so should be good to go. (This is needed before https://github.com/eXist-db/exist/pull/2423 can be merged.)
2. A new release needs to be prepared: increment the version number in `expath-pkg.xml`, release notes added to the `<changelog>` in `repo.xml`, draft a release on github pasting in the release notes and the xar, and publish the xar to the public-repo. I'm happy to do this.

**Comment by adamretter** [comment]
I believe that Wolfgang mentioned that Polymer would be unsupported in Chrome in March. Presumably that also has to be addressed before we release?

**Comment by joewiz** [comment]
@adamretter I updated https://github.com/eXist-db/exist/pull/2423 to note that it shouldn't be merged until https://github.com/eXist-db/existdb-dashboard/issues/16 is resolved.

I'd still welcome any member of @eXist-db/xquery-contributors to merge https://github.com/eXist-db/existdb-dashboard/pull/14.

**Comment by joewiz** [comment]
With https://github.com/eXist-db/existdb-dashboard/pull/14 now merged, the primary question to be resolved regarding the new Dashboard's immediate future is https://github.com/eXist-db/existdb-dashboard/issues/16.
